tools/ci_build/build.py

```diff
@@ -569,6 +570,23 @@ def setup_test_data(build_dir, configs):
                                 src_model_dir], shell=True)
 
 
+def check_all_flags_supported(input_flags):
+    supported = False
+    # No Windows support for now
+    if is_windows():
+        return supported
```

Comment:
Can we make the function name more specific here -- we are not saying that build.py is unsupported on Windows, right?

Reply:
I will change this to only detect supportability of cpu fp16 isa then. The issue with Windows is that MSVC doesn't have support for f16c because of the missing linker flags and some header files, so this will only be available on Linux OS.